working-days/working-days.component.html

```diff
@@ -0,0 +1,66 @@
+<div class="container">
+  <form>
+    <table style="border:0px">
+        <h4>Working Days</h4><span style="padding: 40px"></span>
+      <div class="container" fxLayout="coloumn" fxFlex="80" fxLayoutAlign="left">
+        <tr>
+          <mat-checkbox color="primary">Monday</mat-checkbox>
+        </tr>
+        <tr>
+          <mat-checkbox color="primary">Tuesday</mat-checkbox>
+        </tr>
+        <tr>
+          <mat-checkbox color="primary">Wednesday</mat-checkbox>
+        </tr>
+        <tr>
+          <mat-checkbox color="primary">Thursday</mat-checkbox>
+        </tr>
+        <tr>
+          <mat-checkbox color="primary">Friday</mat-checkbox>
+        </tr>
+        <tr>
+          <mat-checkbox color="primary">Saturday</mat-checkbox>
+        </tr>
+        <tr>
+          <mat-checkbox color="primary">Sunday</mat-checkbox>
+        </tr>
+      </div>
+    </table>
+  </form>   
+</div>
+
+<div class="container" fxLayoutAlign="left">
+  <form>
+    <table style="border:0px">
+      <tr><td><h4>Payment due on<br>Non Working Days</h4></td>
+        <td><span style="padding: 37px"></span>
+      </td>
+        <td><mat-form-field>
+          <select matNativeControl required>
+            <option value="same-day">Same Day</option>
```

Comment:
Values should be fetched and populated by using the appropriate `API`. Don't hardcode.